package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "resin-bundle-resolve",
+  "version": "0.0.1",
+  "description": "Resolve resin-bundles into a docker daemon format",
+  "main": "lib/index.js",
+  "types": "src/index.ts",
+  "scripts": {
+    "start": "",
+    "build": "gulp",
+    "pretest": "tsc tests/tests.ts",
```

Comment:
The problem I was seeing with that was that as I was providing a `rootDir` to `tsc` and then mocha tried to compile, `tsc` was complaining that the tests were not part of `rootDir`. I played around with the different settings for `rootDir` and mocha but this way just seemed to work best. I agree though it's not ideal spamming the built js files. I'll look at getting this sorted properly. It seems that the difference between this and `typed-error` is that `typed-error` specifies the input using the `files` option, and that would become a bit much for larger codebases. Suggestions welcome :)